jira/jira.go

```diff
@@ -123,15 +142,17 @@ func promptForAuth(ctx *ankh.ExecutionContext, retryCount int) (string, string,
 	return providedUsername, providedPassword, nil
 }
 
-func getSummary(ctx *ankh.ExecutionContext, env string) (string, error) {
+func getSummary(ctx *ankh.ExecutionContext, chart *ankh.Chart, envOrContext string) (string, error) {
 	// If format is set, use that
 	format := ctx.AnkhConfig.Jira.SummaryFormat
 	if ctx.Mode == ankh.Rollback {
 		format = ctx.AnkhConfig.Jira.RollbackSummaryFormat
 	}
 
+	chartName := fmt.Sprintf("%v@%v", chart.Name, chart.Version)
+
 	if format != "" {
-		message, err := util.ReplaceFormatVariables(format, ctx.Chart, ctx.DeploymentTag, env)
+		message, err := util.ReplaceFormatVariables(format, chartName, *chart.Tag, envOrContext)
```

Comment:
chartName confused me slightly here because it's the name plus the version. let's rename the local variable to something that captures that fact.